Doc/library/mmap.rst

```diff
@@ -198,6 +198,18 @@ To map anonymous memory, -1 should be passed as the fileno along with the length
       exception is raised when the call failed.
 
 
+   .. method:: madvise(option[, start[, length]])
```

Comment:
It would be good to also document the available `MADV_` constants (not necessarily describe them, but document their presence).
See e.g. how this is done here:
https://docs.python.org/3/library/os.html#os.posix_fadvise
or
https://docs.python.org/3/library/socket.html#constants